### PR TITLE
Add Kudox column-staggered layout.

### PR DESCRIPTION
**Pull request by kumaokobo**
## Description

Add Kudox column-staggered layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kudox/columner/info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                {"label":"LANG2", "x":4.25, "y":4.5},
+                {"label":"GUI", "x":5.25, "y":4.75},
+                {"label":"Space", "x":6.25, "y":5},
+                {"label":"Space", "x":10, "y":5},
+                {"label":"GUI", "x":11, "y":4.75},
+                {"label":"LANG2", "x":12, "y":4.5},
+                {"label":"\u2190", "x":13, "y":4},
+                {"label":"\u2193", "x":14, "y":4.25},
+                {"label":"\u2191", "x":15, "y":4.5},
+                {"label":"\u2192", "x":16, "y":4.5, "w":1.25},
```
Comment (noroadsleft) [inline keyboards/kudox/columner/info.json]: Extra comma here.

```suggestion
                {"label":"\u2192", "x":16, "y":4.5, "w":1.25}
```

Comment (kumaokobo) [reply keyboards/kudox/columner/info.json]: Thank you for suggestion!

**Comment by kumaokobo** [comment]
@noroadsleft Thank you for your reviewing!
I fixed the code pointed.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**